nsxt/policy_common.go

```diff
@@ -235,7 +235,11 @@ func getSecurityPolicyAndGatewayRulesSchema(scopeRequired bool) *schema.Schema {
 					Optional:    true,
 					Default:     false,
 				},
-				"tag": getTagsSchema(),
```

Comment:
I think we need to preserve this tag, and add the other one with name "log_label" as it appears in the UI